### PR TITLE
Revert "further dab fixes" but keeps some sanity for dabwizards

### PR DESCRIPTION
**Pull request by SirSkeleto**
Reverts vgstation-coders/vgstation13#21366

:cl:
 * rscdel: reverts "further dab fixes"

### CODE REVIEW

**Comment by beesil** [comment]
FREE *DAB

**Comment by bathosbathtime** [comment]
Please reintroduce the restriction on wizards, they can not be allowed to dab.

**Comment by SirSkeleto** [comment]
> Please reintroduce the restriction on wizards, they can not be allowed to dab.

:+1:

**Comment by cowboybuttsdrivemenuts** [comment]
merge when

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -213,66 +213,39 @@
...
-		for(var/datum/organ/external/breakthis in H.get_organs(LIMB_LEFT_ARM, LIMB_RIGHT_ARM, LIMB_HEAD))
-			H.apply_damage(50, BRUTE, breakthis)
-			if(!(H.species.anatomy_flags & NO_BONES))
-				breakthis.fracture()
-		H.adjustOxyLoss(max(175 - H.getToxLoss() - H.getFireLoss() - H.getBruteLoss() - H.getOxyLoss(), 0))
-		H.updatehealth()
+	if(iswizard(H))
+		to_chat(user, "<span class='warning'>The Wizard Federation has banned usage of the [key].</span>")
+		return FALSE
+	if(H.stunned || H.restrained())
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/human/emote.dm:229]: Wasn't the intent that you can dab while cuffed but not while unconcious? That isn't what this does.

**Comment by Hoshi-chan** [comment]
If there was ever a time a speedmerge was warranted, it's now.

**Comment by Tokanova** [comment]
everyone loves this PR, the emojicracy has spoken.

**Comment by MadmanMartian** [comment]
Nah

**Pull request closed without merging**